src/utils/accessibility/accessibility.js

```diff
@@ -3,20 +3,32 @@ function isLargeScale(fontSize, isFontBold) {
   return isFontBold ? (fontSize >= 14) : (fontSize >= 18);
 }
 
-function accessibleContrast(accessibilityLevel, fontSize, isFontBold) {
+function fontSizeInPt(fontSize,fontUnitOfMeasure) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+  return (fontUnitOfMeasure === 'pt') ? fontSize : Math.round(fontSize * 0.75);
+}
+
+function fontSizeInPx(fontSize,fontUnitOfMeasure) {
+  // http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head
+  return (fontUnitOfMeasure === 'pt') ? Math.round(fontSize * 1.333) : fontSize;
```

Comment:
This would be easier to digest, and would be more consistent with `fontSizeInPt()`:
```js
return (fontUnitOfMeasure === 'px') ? fontSize : Math.round(fontSize * 1.333);
```